src/pages/learn/ecommerce/build-a-content-management-system-for-an-e-commerce-store-with-nextjs-and-sanity.mdx

```diff
@@ -0,0 +1,425 @@
+
+```bash
+yarn create next-app my-sanity-store
+```
+
+This will create a new Next.js application at `my-sanity-store`.
+
+Once that’s finished installing, you can navigate into your project and start your development server:
+
+```bash
+cd my-sanity-store
+yarn dev
+```
+
+That will load the Next.js app and we’ll be ready to go!
+
+![New Next.js app](https://res.cloudinary.com/dg3gyk0gu/image/upload/v1611073514/next.egghead.io/pages/learn/ecommerce/build-a-content-management-system-for-an-e-commerce-store-with-nextjs-and-sanity/new-next-js-app_2x.png)
+
+[Follow along with the commit!](https://github.com/colbyfayock/my-sanity-store/commit/943530cd199db8a9e6076b66465e7f91a1c7f760)
+
+## Step 1: Creating a new Sanity Studio project
+
+To get started with Sanity, unless you use one of [their Starters](https://create.sanity.io/) which bootstraps an entire project complete with deployment, you need to use the terminal and yarn to create a new project.
+
+While using a Starter would be easier to get to a “finished” state, we’re going to walk through how to do this manually so you can then add it to an existing project or customize it yourself like you might do in a real world project.
+
+> Tip: If this is your first time using Sanity, you’ll notice that there’s not a traditional “Sign Up” page like you typically would expect. Instead, once you create your project, Sanity will give you an opportunity to log in and connect your project to an account.
+
+Inside of your terminal, first install the sanity-cli with:
+
+```bash
+yarn add @sanity/cli --global
+```
+
+Then, inside of the folder you’d like to create your new project, which should be separate from your Next.js project, run:
+
+```bash
+sanity init
+```
+
+If this is your first time logging into Sanity in your terminal, you’ll now be prompted to log in to an account. If you don’t have an account, logging in will set up that account for you with Google, GitHub, or it will give you the opportunity to sign up with an email and password.
+
+Once logged in, Sanity will then prompt you to answer a few questions.
+
+![Sanity Prompts](https://res.cloudinary.com/dg3gyk0gu/image/upload/v1611073515/next.egghead.io/pages/learn/ecommerce/build-a-content-management-system-for-an-e-commerce-store-with-nextjs-and-sanity/sanity-prompts_2x.png)
+
+You can name the project anything you’d like, but it will be used as Sanity is setting up your project
+
+For the dataset configuration, we can stick with the default for now, so you can answer Yes.
+
+Finally, you can customize your output directory or let it use the default, which will create the project in the folder you’re in, based off of your project name.
+
+Once you answer those first few questions, it will ask you to select a project template. If you’re starting from scratch and want to create a custom CMS schema, you may want to use Clean, but we’re going to build an ecommerce project, so let’s select the E-commerce option.
```

Comment:
```suggestion
Once you answer those first few questions, it will ask you to select a project template. If you’re starting from scratch and want to create a custom CMS schema, you may want to use "Clean", but we’re going to build an ecommerce project, so let’s select the "E-commerce" option.
```